e107_admin/auth.php

```diff
@@ -66,9 +66,9 @@
 		// XXX LOGIN AS Temporary solution, we need something smarter, e.g. reserved message stack 'admin' which will be always printed
 		// inside admin area
 		if(e107::getUser()->getSessionDataAs())
-		{ // TODO - lan
+		{ // TODO - lan [logout]?
 			$asuser = e107::getSystemUser(e107::getUser()->getSessionDataAs(), false);
-			e107::getMessage()->addInfo('Successfully logged in as '.($asuser->getId()  ? $asuser->getName().' ('.$asuser->getValue('email').')' : 'unknown'). ' <a href="'.e_ADMIN_ABS.'users.php?mode=main&amp;action=logoutas">[logout]</a>');
+			e107::getMessage()->addInfo(ADLAN_164.' '.($asuser->getId()  ? $asuser->getName().' ('.$asuser->getValue('email').')' : 'unknown'). ' <a href="'.e_ADMIN_ABS.'users.php?mode=main&amp;action=logoutas">[logout]</a>');
 		}
```

Comment:
Done but note there are duplicate messages coming from admin/users.php 
Note that the further instruction message is a little confusing. 

![image](https://cloud.githubusercontent.com/assets/2847651/20871102/9cdd5d74-ba89-11e6-92e4-d6e2e42689ef.png)